### PR TITLE
update README in the main repo and in each package

### PR DESCRIPTION
**Pull request by truongsinh**
- build status and code coverage (changing from my personal repo to zino-app's)
- "all contributors". Btw, https://allcontributors.org/docs/en/bot/overview instead

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -175,6 +175,158 @@
...
+        "tutorial"
+      ]
+    },
+    {
+      "login": "mainawycliffe",
+      "name": "Maina Wycliffe",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/12270550?v=4",
+      "profile": "https://codinglatte.com",
+      "contributions": [
+        "code"
```
Comment (mainawycliffe) [inline .all-contributorsrc]: Also include docs and examples in my contributions

**Comment by mainawycliffe** [comment]
LGTM

**Comment by truongsinh** [comment]
changes approved, sb authorized to merge please do so :D

**Pull request merged**